src/Components/Layout/CleanLayout/CleanLayout.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import "./CleanLayout.scss";
+
+const UnauthenticatedLayout = ({ children }) => {
+  return <div className={`clean-container`}>{children}</div>;
+};
+
+export default UnauthenticatedLayout;
```

Comment:
Name should also be ```CleanLayout```.

Reply:
oh sorry, I totally forgot to change this